CSE/Logic/DataManagement/TaskInit.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Logic.Database;
+using Logic.Metadata;
+using System.Threading;
+using Logic.Models;
+
+namespace Logic.DataManagement
+{
+	public class TaskInit
+	{
+		public Receipt receipt;
+		public Receipt Receipt { get; set; }
+
+		public void OnListInitialized(object source, EventArgs e)
+		{
+			ThreadStart update = new ThreadStart(UpdateThread);
+			Thread thread = new Thread(update);
+		}
+
+		public void UpdateThread()
+		{
+
+			Store store = Store.Iki;
+			DataModel dm = new DataModel();
```

Comment:
I see you don't use dependency injection here, not sure if after merging this pull request errors won't appear as parameterless constructor for DataModel does not exist as far as I know. (The same for Updater etc).